azure-stack/hci/manage/trusted-launch-vm-overview.md

```diff
@@ -34,6 +34,14 @@ Trusted launch is a security type that can be specified when creating Arc VMs on
 > [!NOTE]
 > VM guest boot integrity verification is not available.
 
+## Guidance
+
+- IgvmAgent is a component that is installed on all nodes in the Azure Stack HCI cluster. It enables support for isolated VMs such as Trusted launch Arc VMs for example.
+
+- As part of Trusted launch Arc VM creation, Hyper-V creates VM files on disk to store the VM state. By default, access to those VM files is restricted to host server administrators. Host administrators must ensure that the location where those VM files are stored remains appropriately access-restricted at all times.
+
+- VM live migration network traffic is not encrypted. We strongly recommend that you enable a network layer encryption technology such as SMB encryption or IPsec to protect live migration network traffic.
```

Comment:
Please mind that SMB encryption will prevent use of RDMA. We need a remark box on that. IPsec seems like Overkill. Live mig traffic is in normal cases SMB 3,unless using old TCP compression.